app/src/main/java/com/codesoom/assignment/handler/RequestHandler.java

```diff
@@ -0,0 +1,143 @@
+package com.codesoom.assignment.handler;
+
+import com.codesoom.assignment.dto.TaskDto;
+import com.codesoom.assignment.models.MethodType;
+import com.codesoom.assignment.models.StatusCode;
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RequestHandler implements HttpHandler {
+    private final static List<Task> tasks = new ArrayList<>();
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private final Pattern urlPattern = Pattern.compile("/tasks/(\\d)");
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        MethodType methodType = MethodType.valueOf(exchange.getRequestMethod());
+
+        switch (methodType) {
+            case GET:
+                getTasksProcessor(exchange);
+                break;
+            case POST:
+                postTasksProcessor(exchange);
+                break;
+            case PUT:
+            case PATCH:
+                putTasksProcessor(exchange);
+                break;
+            case DELETE:
+                deleteTasksProcessor(exchange);
+                break;
+            default:
+                throw new IllegalArgumentException("지원되지 않는 METHOD 타입입니다.");
+        }
+    }
+
+    private void getTasksProcessor(HttpExchange exchange) throws IOException {
+        String path = urlPath(exchange);
+        Matcher matcher = urlPattern.matcher(path);
+
+        if (path.equals("/tasks")) {
+            String content = taskToJSON(tasks);
+            sendResponse(exchange, content, StatusCode.OK);
+        }
```

Comment:
```suggestion
        if (path.equals("/tasks")) {
            String content = taskToJSON(tasks);
            sendResponse(exchange, content, StatusCode.OK);
            return
        }
```

이렇게 해주면, 이러한 경우에는 더 이상 밑의 코드를 읽지 않아도 되기 때문에 읽는 사람의 부하가 더 작아질거에요

Reply:
void 메소드라서 return을 생각못했었는데, 리턴타입과 상관없이, 메소드 실행을 종결시킨다는 점에서 좋은것 같습니다!